test/lit/passes/vacuum-gc.wast

```diff
@@ -18,21 +18,21 @@
   ;; CHECK-NEXT:   )
   ;; CHECK-NEXT:  )
   ;; CHECK-NEXT:  (drop
-  ;; CHECK-NEXT:   (ref.cast i31
+  ;; CHECK-NEXT:   (ref.cast null i31
   ;; CHECK-NEXT:    (local.get $x)
   ;; CHECK-NEXT:   )
   ;; CHECK-NEXT:  )
   ;; CHECK-NEXT: )
   (func $drop-ref-as (param $x anyref)
-    ;; Without -tnh, we must assume all ref_as* can have a trap effect, and so
+    ;; Without -tnh, we must assume all casts can have a trap effect, and so
     ;; we cannot remove anything here.
     (drop
       (ref.as_non_null
         (local.get $x)
       )
     )
     (drop
-      (ref.as_i31
+      (ref.cast null i31
```

Comment:
Why change the nullability?
```suggestion
      (ref.cast i31
```

Reply:
(this also happens in at least one other test below, so maybe I'm missing something...)

Reply:
I don't think I remembered whether the result of `ref.as_i31` was supposed to be nullable or not, so I think I just took a guess 😅 It doesn't seem to have affected the test output unexpectedly, so I'm not sure it's worth changing. WDYT?

Reply:
Just for some rationale here: In the future when I update the text format, this will be able to be `ref.cast i31ref` rather than `ref.cast (ref i31)`, so it's a tiny simplification.